update-serve-file.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
@staabm I'll describe the situation:
We have a package `wp-web-push` which depends on package `wp-web-app-manifest-generator` and package `wp_serve_file`. Package `wp-web-app-manifest-generator` depends on package `wp_serve_file`.
In package `wp-web-push` files, we can load package `wp-web-app-manifest-generator` and package `wp_serve_file` files easily by calling `require_once('vendor/marco-c/wp-web-app-manifest-generator/something.php')` or `require_once('vendor/marco-c/wp_serve_file/something.php')`.
In package `wp-web-app-manifest-generator` files, how do we import package `wp_serve_file` files? Composer installs the dependencies only in the top-level directory.

So, in `wp-web-push`, we have something like this:
```
wp-web-push/
├── lang
├── lib
│   ├── js
│   └── style
└── vendor
    ├── marco-c
    │   ├── wp_serve_file
    │   └── wp-web-app-manifest-generator
    └── mozilla
        └── wp-sw-manager
            └── lib
                └── js
```

In `wp-web-app-manifest-generator`, we're using a very hacky `require_once('../../marco-c/wp_serve_file/class-wp-serve-file.php');`.